### PR TITLE
Python3 update passwd remote token tests

### PR DESCRIPTION
**Pull request by plettich**
Working on #676

### CODE REVIEW

File: privacyidea/lib/tokens/remotetoken.py [hunk 1]
```diff
@@ -218,34 +218,24 @@ def check_otp(self, otpval, counter=None, window=None, options=None):
...
         """
         otp_count = -1
-        otpval = otpval.encode("utf-8")
 
         remoteServer = self.get_tokeninfo("remote.server") or ""
-        remoteServer = remoteServer.encode("utf-8")
 
         # in preparation of the ability to relocate privacyidea urls,
         # we introduce the remote url path
         remotePath = self.get_tokeninfo("remote.path") or ""
```
Comment (fredreichbier) [inline privacyidea/lib/tokens/remotetoken.py]: With this change, ``removePath`` is not stripped anymore. I'm not sure it's necessary?

Comment (plettich) [reply privacyidea/lib/tokens/remotetoken.py]: Oh, totally missed that. Will ad a test for remote.path as well.

Comment (cornelinux) [reply privacyidea/lib/tokens/remotetoken.py]: The background is, that the remote privacyIDEA server could run e.g. at https://remote.example.com/pi2 while the originial privacyIDEA server would run at https://local.exampl.com/

Comment (plettich) [reply privacyidea/lib/tokens/remotetoken.py]: Fixed in 6141b198

Comment (fredreichbier) [reply privacyidea/lib/tokens/remotetoken.py]: :+1:

**Pull request merged**